### ISSUE TITLE
Relentless Assault only fired once

### ISSUE THREAD

**Issue by AdamPhilpotMDX** [jeremylarner/keyteki#100]
Hello Jeremy,

Great work on the game, and thanks for all of your effort.

I called Brobnar as active house and had one ready Dis creature (Guardian Demon).  I played Troll.  Then played Relentless Assault.  I selected the ready Dis creature and fought Lady Maxena who had extra armour from Bulwark.  Guardian Demon was destroyed and there was no option to select another creature to fight.

Was I supposed to choose all of my creatures to fight first before clicking done?  I assumed that I would 'go through the motions' one at a time, as instructed by the card.  i.e. select creature, click done, select enemy creature with whom to fight, select next creature, click done etc.

My opponent was understanding and thankfully accepted manual mode to ready my Troll and it fought and destroyed Lady Maxena.

Thanks for the excellent work, best wishes, 

Adam

**Comment by jeremylarner** [comment]
Yeah, you need to click the three creatures you want to fight with when you play the ability.  I'm not sure if that's correct (e.g. can I play Relentless Assault, fight with Overlord Greking, kill something, take control of it, then use that creature as my second for RA?)

**Comment by neffmallon** [comment]
> Play: Ready and fight with up to 3 different friendly creatures, one at a time.

Each fight should be resolved in turn, right? You ready and fight with a creature, resolve any "fight" text, then do the same with a different friendly creature, then with another one. If the "fight" text gives you access to another friendly creature, then you can use that one if you want, right?
